configs/DAKEFPVF722_HD/config.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        DAKEFPVF722_HD
+#define MANUFACTURER_ID   DAKE
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#ifndef USE_CAMERA_CONTROL
+#define USE_CAMERA_CONTROL
+#endif
+
+#define BEEPER_PIN           PC3
+#define LED_STRIP_PIN        PA0
+#define MOTOR1_PIN           PA1
+#define MOTOR2_PIN           PA2
+#define MOTOR3_PIN           PA3
+#define MOTOR4_PIN           PA8
+#define MOTOR5_PIN           PA9
+#define MOTOR6_PIN           PA10
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
```

Comment:
- Only 4 motor outputs allowed
- PA0 would be perfect candidate for motor 4 (@ot0tot please check)

https://betaflight.com/docs/development/manufacturer/config-target-guidance

Reply:
Okay, I got it.